<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateRangeDialog</class>
 <widget class="QDialog" name="DateRangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор периода</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="start_date_layout">
     <item>
      <widget class="QLabel" name="start_date_label">
       <property name="text">
        <string>Начальная дата:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="start_date">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="end_date_layout">
     <item>
      <widget class="QLabel" name="end_date_label">
       <property name="text">
        <string>Конечная дата:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="end_date">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout">
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_button">
       <property name="text">
        <string>Выйти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
